.eslintrc

```diff
@@ -20,7 +20,7 @@
   },
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaVersion": 8,
+    "ecmaVersion": 9,
```

Comment:
It's not necessary to bump the ECMAVersion eslint tries to parse, since it didn't change.

Reply:
Thanks, I reverted that file to use 8.